packages/agoric-cli/tools/getting-started.js

```diff
@@ -181,8 +181,8 @@ export const gettingStartedWorkflowTest = async (t, options = {}) => {
         if (currentHeight > lastKnownBlockHeight) {
           const earlierHeight = lastKnownBlockHeight;
           lastKnownBlockHeight = currentHeight;
-          if (earlierHeight > 0n && currentHeight > earlierHeight) {
-            // We've had at least one block produced.
+          if (earlierHeight > 2n && currentHeight > earlierHeight) {
+            // We've had at least 3 blocks produced.
```

Comment:
How is waiting for three blocks fix the account seq mismatch issue? Isn't it due to parallel jobs doing tx with same account?

Reply:
Hmm right. So As far as I know, this tx sequence is linked with account/wallet as well. Our tests use user1 (install-bundles.sh), so about governance wallet reansactions, their sequence number shouldn't disturb user1, right? Not sure just thnking out loud.

Reply:
`user1` is involved in installing bundles. But for the contract core eval to be passed, I reckon governance wallets take part in it via voting on it. I think that phase leads to account sequence mismatch.

Reply:
Yes. Mujahid and I ran it several times. Did not encounter any flakes or failures.

Reply:
https://github.com/Agoric/agoric-sdk/actions/workflows/integration.yml?query=branch%3Ars-fix-getting-started-flakes